mkosi/installer/moss.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+from collections.abc import Sequence
+
+from mkosi.bubblewrap import apivfs_cmd, bwrap
+from mkosi.context import Context
+from mkosi.types import PathString
+from mkosi.util import sort_packages
+
+
+def moss_cmd(context: Context) -> list[PathString]:
+    cmdline: list[PathString] = [
+        "moss",
+        "--directory",
+        f"{context.root}",
+        "--yes-all",
+    ]
+
+    return cmdline
```

Comment:
No need for the extra variable and annotation
```suggestion
    return [
        "moss",
        "--directory",
        f"{context.root}",
        "--yes-all",
    ]
```

The same style is used e.g. in pacman.py

Reply:
It might also make sense for moss_cmd to take a command argument, see apt_cmd, which would make some of the list additions further down unnecessary.